.github/workflows/deploy-pages.yml

```
@@ -6,6 +7,18 @@ on:
         required: false
         default: "./docs"
         type: string
+      node-version:
+        required: false
+        default: "22"
+        type: string
+      build-cmd:
+        required: false
+        default: "build"
+        type: string
+      dist-path:
+        required: false
+        default: ".vitepress/dist"
```

hier einfach nur dist. wwil refarch team hat es wieder geändert
```suggestion
        default: "dist"
```

---

@hupling how do you come to this? In my opinion that's not correct and it was right before